### PR TITLE
Syntax Highlighting, Example Hello World policy

### LINKED ISSUE #65: Rego Policy Improvements
Look into adding syntax highlighting wherever we show rego policy code

Links from Robert:
https://prismjs.com/
https://jcguest90.medium.com/reacttext-editor-with-syntax-highlighting-9ae15d93cac0
https://www.npmjs.com/package/react-simple-code-editor?activeTab=readme

--- 
Add a "Hello World" policy that outlines our expectations of a policy to get people started.

---
Improve styling of errors that are returned when validating a policy. Currently they are just one big line of the array of errors, but this could be styled to be easier to read.

### PR DESCRIPTION
**Pull request by pickjasmine**
Closes #65 

* Adds syntax highlighting to wherever code is shown, including the text field that you type your rego policy code into. 
* Adds an example policy that gets auto-populated to the create policy form.

---
_some things to note with the addition of syntax highlighting..._

The highlighter sometimes takes a moment to show the colors (viewing the occurrence JSON ,for example). I have added this to an issue to address later since it's not really breaking any functionality, it's just a weird experience.

With the change in to highlight code, the policy playground now shows the entire rego policy code, which is not a good experience if you have a lengthy policy. I plan on addressing this issue in my next PR.

The `prism.js` file included this PR is the code to do syntax highlighting. There is an npm package for Prism, but the latest release does not include the ability to parse rego. Once a new release of Prism and the [babel plugin](https://github.com/mAAdhaTTah/babel-plugin-prismjs) are cut, we can switch over to pulling down those dependencies instead of including them directly. The file is small and the parsing shouldn't change so not a huge deal, in my opinion. I have ignored the Prism file for linting and coverage.

### CODE REVIEW

File: prism/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 The Rode Authors
```
Comment (pickjasmine) [inline prism/LICENSE:3]: I'm not sure if I should put `The Rode Authors` here or if this should be a reference to the original creators of Prism?

File: test/components/Code.spec.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+import Code from "components/Code";
+
+describe("Code", () => {
+  let code;
+
+  beforeEach(() => {
+    code = chance.string();
+
+    Prism.highlightAll = jest.fn();
```
Comment (alexashley) [inline test/components/Code.spec.js:29]: Is this still needed if it's in the global jest setup?

File: components/Code.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import Prism from "prism/prism";
+import styles from "styles/modules/Typography.module.scss";
+
+// TODO: how to prevent non-highlighted code before highlight appears?
+const Code = (props) => {
+  const { code, language, className = "", ...otherProps } = props;
+
+  useEffect(() => {
+    Prism.highlightAll();
+  }, []);
```
Comment (alexashley) [inline components/Code.js:28]: Should the dependency array include `code`/`language`? They're not used directly, but if they change would this need to run again?

File: prism/prism.js [hunk 4]
```diff
@@ -0,0 +1,757 @@
+/**
+ * Copyright 2021 The Rode Authors
```
Comment (alexashley) [inline prism/prism.js:2]: I don't think we want our copyright header in this one

**Review by alexashley — COMMENTED** [review]
With the light theme, the text contrast with a background is a little difficult to read IMO

Screen Shot 2021-04-28 at 4 30 31 PM

Otherwise, LGTM, just had a few questions

File: components/Code.js (hunk 3 above)
Comment (pickjasmine) [reply components/Code.js:28]: Makes sense to me. The code/language shouldn't change on any of the current pages but it could in the future.

**Comment by pickjasmine** [comment]
> With the light theme, the text contrast with a background is a little difficult to read IMO
> 
> Screen Shot 2021-04-28 at 4 30 31 PM
> 
> Otherwise, LGTM, just had a few questions

I updated the colors for both themes a bit so now there should be a distinction between comments and regular code. Let me know if you still think the contrast is a bit too low.

File: prism/prism.js (hunk 4 above)
Comment (pickjasmine) [reply prism/prism.js]: I removed our copyright but now the license check is failing.. Looks like there [isn't a way to ignore files](https://github.com/google/addlicense/issues/53).

Comment (pickjasmine) [reply prism/prism.js]: Looks like someone opened a [PR](https://github.com/google/addlicense/pull/73) to add the ability to ignore.  For now I'm going to add our copyright header back in so I can keep moving on. I'll make an issue to address this in the future.

**Review by mrparkers — APPROVED** [review]

**Pull request merged**